lms/static/sass/search/_search.scss

```diff
@@ -124,7 +123,7 @@
 }
 
 .courseware-search-bar {
-  box-shadow: 0 1px 0 $white inset, 0 -1px 0 $shadow-l1 inset;
+   width: flex-grid(7);
```

Comment:
Not able to see these changes https://github.com/edx/edx-platform/pull/8082/files#diff-2ddc5e8947efe274084196d8b90b41b2R221